app/controllers/wikis_controller.rb

```diff
@@ -3,28 +3,27 @@
 class WikisController < ApplicationController
   def index
     @wikis = policy_scope(Wiki).paginate(page: params[:page], per_page: 20)
-    authorize Wiki
   end
 
   def show
     @wiki = Wiki.find(params[:id])
-    @user = current_user
     authorize @wiki
-    @collaborators = Collaborator.where(wiki_id: @wiki.id)
-    @collaborator = @wiki.collaborators.build
+    @user = current_user
+    @collaborators = @wiki.collaborators
```

Comment:
Higher-level way of doing `Collaborator.where(wiki_id: @wiki.id)`. Rails will then cache the results.